fourkitchens/sous-drupal-project
# Drupal9.1 compatibility

## Purpose:
- Drupal9.1 compatibility

## Testing:
- Pull down this branch
- If you are using an existing clone of the project check your settings.php file for `$databases['default']['default']` and if necessary update the lando config from referencing drupal8 to drupal9.
- `lando composer install`
- Verify that composer installs without error
